.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release to production
+on:
+  repository_dispatch:
+    types: [prod-release]
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+
+      - name: Set NPM_TOKEN
+        run: echo "NPM_TOKEN=${{ secrets.NPM_TOKEN }}" >> $GITHUB_ENV
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build package
+        run: npm run build
+
+      - name: Release
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+          ENVIRONMENT: ${{ github.ref }}
+        run: npm publish --workspace client --access public
```

Comment:
We'll have to update the version manually, just making sure you take a note of this

Reply:
Noted.